### ISSUE TITLE
learning rate schedule?

### ISSUE THREAD

**Issue by RudyChin** [kuangliu/torchcv#1]
Hi,
In terms of the mAP you are getting for SSD, I wonder how many epochs do you train the network? (I find 200 in the code)
Also, I didn't find learning rate schedule policy in the code, does that mean you train the network for 200 epochs with learning rate fixed?

**Comment by kuangliu** [comment]
I use:
```
1e-3 for epoch [0,100)
1e-4 for epoch [100,120)
1e-5 for epoch [120,140)
```
I change `lr` by hand. 

UPDATE: I checked my training log, 1e-3 for 100 epochs, not 200.

**Comment by RudyChin** [comment]
Thank you!
